Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/security-emailthreatsubmission-list.md

---
title: "List emailThreatSubmissions"
description: "Get a list of the emailThreatSubmission objects and their properties."
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

# List emailThreatSubmissions
Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Get a list of the [emailThreatSubmission](../resources/security-emailthreatsubmission.md) objects and their properties.

## Permissions

> One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

Reviewer: One of the following permissions is required to call this API. To learn more permissions, including how to choose one, see [Permissions](/graph/permissions-reference).

> One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

Reviewer: ```suggestion
One of the following permissions is required to call this API. To learn more permissions, including how to choose one, see [Permissions](/graph/permissions-reference).
```

| Server-side pagination |     ✓     | `@odata.nextLink`                                            |       |
| Filter                 |     ✓     | `/emailThreats?$filter=emailSubject eq 'value'`              |       |
| Filter                 |     ✓     | `/emailThreats?$filter=sender eq 'value'`                    |       |
| Filter                 |     ✓     | `/emailThreats?$filter=id eq 'value'`                        |       |
| Filter                 |     ✓     | `/emailThreats?$filter=createdDateTime ge 'value' and createdDateTime lt 'value'` |       |
| Filter                 |     ✓     | `/emailThreats?$filter=category eq 'value'`                  |       |
| Filter                 |     ✓     | `/emailThreats?$filter=createdBy eq 'value'`                 |       |
| Filter                 |     ✓     | `/emailThreats?$filter=status eq 'value'`                    |       |
| Filter                 |     ✓     | `/emailThreats?$filter=source eq 'value'`                    |       |

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|

## Request body

> Do not supply a request body for this method.

Reviewer: ```suggestion
A request body is not required for this method. 
```

| Pattern                | Supported | Syntax                                                       | Notes |
| ---------------------- | :-------: | ------------------------------------------------------------ | ----- |
| Server-side pagination |     ✓     | `@odata.nextLink`                                            |       |
| Filter                 |     ✓     | `/emailThreats?$filter=emailSubject eq 'value'`              |       |
| Filter                 |     ✓     | `/emailThreats?$filter=sender eq 'value'`                    |       |
| Filter                 |     ✓     | `/emailThreats?$filter=id eq 'value'`                        |       |
| Filter                 |     ✓     | `/emailThreats?$filter=createdDateTime ge 'value' and createdDateTime lt 'value'` |       |
| Filter                 |     ✓     | `/emailThreats?$filter=category eq 'value'`                  |       |
| Filter                 |     ✓     | `/emailThreats?$filter=createdBy eq 'value'`                 |       |
| Filter                 |     ✓     | `/emailThreats?$filter=status eq 'value'`                    |       |

> | Filter                 |     ✓     | `/emailThreats?$filter=source eq 'value'`                    |       |

Reviewer 2: The Supported and Notes columns are not really meeting any function in the doc. On the other hand, please let me take this to the rest of the team to decide the best way to display the optional query parameters.

> | Pattern                | Supported | Syntax                                                       | Notes |
> | ---------------------- | :-------: | ------------------------------------------------------------ | ----- |
> | Server-side pagination |     ✓     | `@odata.nextLink`                                            |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=emailSubject eq 'value'`              |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=sender eq 'value'`                    |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=id eq 'value'`                        |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=createdDateTime ge 'value' and createdDateTime lt 'value'` |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=category eq 'value'`                  |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=createdBy eq 'value'`                 |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=status eq 'value'`                    |       |
> | Filter                 |     ✓     | `/emailThreats?$filter=source eq 'value'`                    |       |

Reviewer 2: ```suggestion
| Pattern                | Syntax                                                       | 
| ---------------------- | ------------------------------------------------------------ |
| Server-side pagination | `@odata.nextLink`                                            | 
| Filter                 | `/emailThreats?$filter=emailSubject eq 'value'`              | 
| Filter                 | `/emailThreats?$filter=sender eq 'value'`                    | 
| Filter                 | `/emailThreats?$filter=id eq 'value'`                        | 
| Filter                 | `/emailThreats?$filter=createdDateTime ge 'value' and createdDateTime lt 'value'` |
| Filter                 | `/emailThreats?$filter=category eq 'value'`                  | 
| Filter                 | `/emailThreats?$filter=createdBy eq 'value'`                 |
| Filter                 | `/emailThreats?$filter=status eq 'value'`                    | 
| Filter                 | `/emailThreats?$filter=source eq 'value'`                    | 
```

---
title: "List emailThreatSubmissions"
description: "Get a list of the emailThreatSubmission objects and their properties."
author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: apiPageType
---

# List emailThreatSubmissions
Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Get a list of the [emailThreatSubmission](../resources/security-emailthreatsubmission.md) objects and their properties.

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|ThreatSubmission.Read, ThreatSubmission.ReadWrite, ThreatSubmission.Read.All, ThreatSubmission.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported|
|Application|ThreatSubmission.Read.All, ThreatSubmission.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
GET /security/threatSubmission/emailThreats
```

## Optional query parameters

> 
> This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
> 
> | Pattern                | Syntax                                                       | 
> | ---------------------- | ------------------------------------------------------------ |
> | Server-side pagination | `@odata.nextLink`                                            | 
> | Filter                 | `/emailThreats?$filter=emailSubject eq 'value'`              | 
> | Filter                 | `/emailThreats?$filter=sender eq 'value'`                    | 
> | Filter                 | `/emailThreats?$filter=id eq 'value'`                        | 
> | Filter                 | `/emailThreats?$filter=createdDateTime ge 'value' and createdDateTime lt 'value'` |
> | Filter                 | `/emailThreats?$filter=category eq 'value'`                  | 
> | Filter                 | `/emailThreats?$filter=createdBy eq 'value'`                 |
> | Filter                 | `/emailThreats?$filter=status eq 'value'`                    | 
> | Filter                 | `/emailThreats?$filter=source eq 'value'`                    | 

Author: ```suggestion
This method supports `$filter`, `$top`, `$skipToken` and `$count` to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
```
